Docker-client.md

```diff
@@ -21,7 +21,45 @@ Note that `nodeSettings.cloudProvider` should be `local`.
 
 ```sh
 docker run -it --rm \
-    --env-file .env \
     -v $(pwd):/airnode/out \
+    --restart unless-stopped
     api3/airnode-client:pre-alpha
 ```
+
+5. The docker image comes with a [healthcheck status](https://docs.docker.com/engine/reference/builder/#healthcheck) that can be used to monitor the health of the `airnode-client`. If the container becomes unhealthy due to any reason, you should view the logs(in case of any errors) and then restart the container. (Note: healthy status of the container does not indicate that there is no error while running the `airnode-client`, it just ensures that the `airnode-client` is being executed every minute and not halting. It is advisable to view the logs to check the actual status).
+
+6. If you want the `airnode-client` to auto-restart incase of unhealthy status then you can run the following [image](https://github.com/willfarrell/docker-autoheal) that checks for unhealthy containers and restarts them.
```

Comment:
I thought automatic restarts could be implemented without an external dependency with something like https://stackoverflow.com/a/61956475/14558682
but this probably is not the right approach because whatever orchestrates the container is supposed to restart it.